shell/platform/fuchsia/flutter/component.cc

```diff
@@ -454,8 +458,6 @@ Application::Application(
     // false to have the error and stack trace printed in the logs.
     return false;
   };
-
-  AttemptVMLaunchWithCurrentSettings(settings_);
```

Comment:
why is this no longer needed?

Reply:
The Dart VM is process-wide, and each reference to it is tracked.

This code starts up the VM early, before any flutter application is launched, but then immediately drops its VM ref.  Normally this would immediately shutdown the VM, but we set leak_vm to true in our settings_ (the default).  Basically this code is a new without a corresponding delete, and the VM leaks